src/rust/src/file_writer.rs

```diff
@@ -84,7 +84,7 @@ impl<'a> FgbWriter<'a> {
 
         cfgfn(&mut fbb, &mut header_args);
 
-        let mut feat_writer = FeatureWriter::new();
+        let mut feat_writer = FeatureWriter::new(header_args.geometry_type, true);
```

Comment:
I'm surprised by defaulting `force_multi` to true here. For example, that's the opposite of what ogr2ogr does, right?

Should this be user configurable, maybe via the cfgfn?